eng/pipelines/common/ui-tests-compatibility-steps.yml

```diff
@@ -44,7 +44,16 @@ steps:
   - pwsh: ./build.ps1 -Script eng/devices/${{ parameters.platform }}.cake --target=cg-uitest --project="${{ parameters.path }}" --appproject="${{ parameters.app }}" --device="${{ parameters.device }}" --apiversion="${{ parameters.version }}" --configuration="${{ parameters.configuration }}" --results="$(TestResultsDirectory)" --binlog="$(LogDirectory)" ${{ parameters.cakeArgs }} --verbosity=diagnostic --where="${{ parameters.where }}"
     displayName: $(Agent.JobName)
     # retryCountOnTaskFailure: 2
-  
+
+  - bash: |
+      rm -r $HOME/Library/Logs/CoreSimulator/*
+      rm -r $HOME/Library/Logs/DiagnosticReports/*
+      zip -9r "$(LogDirectory)/artifacts/CoreSimulatorLog.zip" "$HOME/Library/Logs/CoreSimulator/"
+      zip -9r "$(LogDirectory)/DiagnosticReports.zip" "$HOME/Library/Logs/DiagnosticReports/"
```

Comment:
do we need to suffix a random string so we don't overwrite things on multiple runs?

Reply:
we probably do. Maybe the date time stuff? yyyyMMddHHmmss

Reply:
@PureWeen  we need the hours and mins. A retry will typically happen in the same hour.

Reply:
yea, I think you caught me between pushes :-) 
Should be good now